wiki/Rules/es.md

```diff
@@ -1,76 +1,73 @@
+  2.  **Deja que los moderadores se encarguen de los problemas en el chat.** Puedes indicarles de cualquier problema con el comando `!report` y ellos se encargaran del resto.
+  3.  **Si alguien ha sido advertido, silenciado o restringido, déjalos y no comiences a hablar de ellos.** Si alguien está restringido, su caso solo les incumbe a ellos y a la administración del juego, a nadie más!
+  4.  **Anima el juego limpio y no prestes atención a tramposos o usuarios tóxicos.** Éstas personas desafortunadamente existen y el prestarles atención sólo los incentiva a continuar o incluso provocar a otros a actuar al igual que ellos. Repórtalos a través del comando de chat `!report` o con los botones en su página de perfil y no te molestes con ellos.
+  5.  **Mantén el chat limpio y apropiado para una comunidad para todas las edades, evitando temas de contenido inapropiado (NSFW) siempre que sea posible.** Contenido inapropiado claramente no es apropiado para una comunidad para todas las edades. Puedes hacer lo que quieras en los chats privados (con mesura claro), pero mantén los canales públicos limpios por el bien de todos.
+  6.  **Sé consciente de lo que compartes con otros, especialmente si el contenido es potencialmente perturbador o de naturaleza extremádamente gráfica.** Si tienes que pensarlo dos veces sobre si algo es o no apropiado de enlazar, probablemente no está bien. Evita cosas con luces rápidas y parpadeantes, gore, imágenes de abuso u cualquier otro contenido muy gráfico.
+  7.  **Mantén los enlaces a sitios legales y adecuados.** Sitios de transmiciones piratas , y cualquier cosa relacionada a piratería, como programas pirateados, de cualquier tipo no está bien y no los queremos en nuestro chat (sitios oficialmente licenciados como Animelab y Crunchyroll están bien). Promocionar enlaces de Twitch para otros videojuegos está bien, pero SOLO en `#videogames`.
+  8.  **No promociones servidores de chat de terceros de ningún tipo, incluyendo servidores/grupos de Discord, Teamspeak o Skype.** El chat de osu! es principalmente sobre *osu!* y no sobre compartir links a tus servidores personales. Hazlo a través de mensajes privados (MP) si es que tienes que hacerlo.
+  9.  **En los canales de idiomas específicos, debes hablar el idioma dominante (inglés en `#english`, tailandés en `#thai`, etc) si es que estás participando en alguna discusión.** En el caso que el lenguaje de un canal no sea explícito (o inferido por el nombre del canal, como en tailandés para `#thai` u holandés para `#dutch`), este siempre será inglés. Los canales #lobby y #help en cualquier lenguaje, a diferencia de los otros canales. *Miembros del [personal de moderación](https://osu.ppy.sh/help/wiki/People/Global_Moderation_Team) no necesitan seguir esta regla, cuando hacen su trabajo.*
```

Comment:
change this link `[personal de moderación](https://osu.ppy.sh/help/wiki/People/Global_Moderation_Team)` to
`[personal de moderación](/wiki/People/Global_Moderation_Team)`